simulariumio/file_converter.py

```diff
@@ -44,6 +42,7 @@ def update_trajectory_info_version(self, data: Dict[str, Any]) -> Dict[str, Any]
         Update the trajectory info block
         to match the current version
         """
+        original_version = int(data["trajectoryInfo"]["version"])
         if int(data["trajectoryInfo"]["version"]) == 1:
```

Comment:
line 46 should use `original_version` now

Reply:
I can set it up differently, but these loops are not if else, so if the version starts at v1, it will get updated to v2 in the first loop, then v2 -> v3 in the next loop, etc. So it needs to keep checking the current version, not the cached original version. 

I'm only storing the original version here so I can report out which version it started from at the end. I changed it to just report that at the beginning of the function to be less confusing. I would get rid of the `original_version` variable completely but I can't embed `int(data["trajectoryInfo"]["version"])` in the f-string. I could also use `original_version` for the first loop logic, but unless I increment it in that loop, I'll need to keep checking the current version in the following `if` loops, not the original version.

Reply:
i see, then my other suggestion is to stash `trajectory_info = data["trajectory_info"]` very early on.  It just gets to be a whole lot of string lookups where you could just do it once and it makes the code a little cleaner.